packages/client/src/ducks/api.ts

```diff
@@ -21,9 +21,49 @@ export const api = createApi({
       transformResponse: (response: { documents: UserDocument[] }) =>
         response.documents,
     }),
+    loginUser: builder.mutation<User, void>({
+      query:() => ({
+        url: "/login",
+        method: "POST",
+      }),
+    }),
+    updatePassword: builder.mutation<User, void>({
+      query: () => ({
+        url: "/users/update_password",
+        method: "PUT",
+      }),
+    }),
+    createUser: builder.mutation<User, void>({
+      query: () => ({
+        url: "/user/create_user",
+        method: "POST",
+      }),
+      transformResponse: (response: { user: User }) => response.user,
+    }),
+    selectUsers: builder.query<User[], void>({
+      query: () => ({
+        url: "/users",
+        method: "GET",
+      }),
+      transformResponse: (response: { users: User[] }) => response.users,
+    }),
+    getMe: builder.query<User, void>({
+      query: () => ({
+        url: "/me",
+        method: "GET",
+      }),
+      transformResponse: (response: { user: User }) => response.user,
+    }),
```

Comment:
I like the idea of keeping these in redux. But its complicated and (currently) unused, lets move that to its own PR